lms/djangoapps/verify_student/views.py

```diff
@@ -855,6 +855,46 @@ def submit_photos_for_verification(request):
     return HttpResponse(200)
 
 
+def _get_location_id(photo_verification):
+    try:
+        ver_status = VerificationStatus.objects.filter(checkpoint__photo_verification=photo_verification).latest()
+        return ver_status.location_id
+    except VerificationStatus.ObjectDoesNotExist:
+        return ""
+
+
+def _get_user_attempts(course_key, user_id, related_assessment):
+
+    return VerificationStatus.objects.filter(
+        user_id=user_id,
+        checkpoint__course_id=course_key,
+        checkpoint__checkpoint_name=related_assessment,
+        status="submitted"
+    ).count()
```

Comment:
Would it make sense to make these helpers class methods on the `VerificationStatus` model?